src/components/common/table/table.js

```diff
@@ -2,21 +2,44 @@ import React, { Component } from 'react'
 import './table.scss';
 import moment from 'moment';
 import Hoc from '../../../services/auxilary';
+import Modal from 'react-responsive-modal';
+import Form from 'components/form/form';
+import { connect } from 'react-redux';
+import { addFeedbackACreator, getFeedbacksACreator } from '../../../actions/projectsActions';
+import Spinner from 'components/common/spinner/spinner';
+import Axios from 'axios';
 class Table extends Component{
     state = {
         trs: [], 
         currentTrs: [],
-        currentOpenedRowId: null
+        currentOpenedRowId: null,
+        opinionModal: false,
+        modalType: false,
+        feedbackItems: [
+            {title: "Opinia", type: "text", placeholder: "dodaj opinie o pracowniku...", 
```

Comment:
This value im component's is required because of detecting which row of table user clicked. But right mów after your sugestion i think better aproach Will when i put index if row im func param.